src/pages/components/button/usage.mdx

```diff
@@ -542,107 +539,805 @@ recommended action labels.
+| -------------- | -------------------------------------------------- | -------------------- |
+| Create/Add     | <Add size={16} aria-label="Add" />                 | Add                  |
+| Edit           | <Edit size={16} aria-label="Edit"/>                | Edit                 |
+| Copy           | <Copy size={16} aria-label="Copy"/>                | Copy                 |
+| Delete         | <TrashCan size={16} aria-label="Trash can"/>       | Trash can            |
+| Remove         | <SubtractAlt size={16} aria-label="Subtract alt"/> | Subtract alt         |
+| Export         | <Export size={16} aria-label="Export"/>            | Export               |
+| Upload         | <Upload size={16} aria-label="Upload"/>            | Upload               |
+| Download       | <Download size={16} aria-label="Download"/>        | Download             |
+| Play/Start     | <Play size={16} aria-label="Play"/>                | Play                 |
+| Pause          | <Pause size={16} aria-label="Pause"/>              | Pause                |
+| Stop           | <StopOutline size={16} aria-label="Stop outline"/> | Stop outline         |
+| Refresh        | <Restart size={16} aria-label="Restart"/>          | Restart              |
+
+Do you think a universal action with a clearly defined icon is missing? Let us
+know [here](https://github.com/carbon-design-system/carbon/issues/new/choose).
+
+Icons that are not in this list can be used in buttons, as long as the icon
+clearly conveys the intended action. To determine the expected use of an icon,
+check its name in the
+[Carbon icon library](https://carbondesignsystem.com/elements/icons/library/).
+
+#### Do not use a defined icon to represent a different universal action
+
+Using icons in the above list for other actions can confuse the expected outcome
+and experience.
+
+<DoDontRow>
+  <DoDont type="do" caption="Do use a defined icon to represent a universal action.">
+
+![Do use a defined icon to represent a universal action.](images/button_usage_button_with_icon_5.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not add an icon for an action that would not be associated with it.">
+
+![Do not add an icon for an action that would not be associated with it.](images/button_usage_button_with_icon_6.png)
+
+  </DoDont>
+</DoDontRow>
+
+#### Launch icon
+
+The launch icon should be used on any call to action that launches the user into
+another tab (whether the content of the new tab is part of the same product or
+an entirely separate web resource). Buttons and links requiring the launch icon
+are often found in the UI left navigation area, side panel, cards, and modals.
+The target destination of the launch action should be made clear to the user
+through the button or link label and the surrounding context. For more guidance,
+see [this section](https://pages.github.ibm.com/carbon/ibm-products/guidelines/content/navigation-labels/) 
```

Comment:
For this link, can we be more specific instead of saying "this section"? It also link to an internal site so we need to put "_For IBMers only:_" at the beginning of the section